### PR TITLE
[13.0][MIG]Migrated report_hotel_reservation.

### PR DESCRIPTION
**Pull request by dhruvitaSerpentCS**

### CODE REVIEW

File: report_hotel_reservation/README.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fvertical--hotel-lightgray.png?logo=github
+    :target: https://github.com/OCA/vertical-hotel/tree/12.0/report_hotel_reservation
```
Comment (Trivedi-Vacha-SerpentCS) [inline report_hotel_reservation/README.rst]: 13.0

File: report_hotel_reservation/README.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+~~~~~~~~~~~~
+
+* Odoo Community Association (OCA)
+* Serpent Consulting Services Pvt. Ltd.
+* Odoo S.A.
+
+Maintainers
+~~~~~~~~~~~
+
+This module is maintained by the OCA.
```
Comment (Trivedi-Vacha-SerpentCS) [inline report_hotel_reservation/README.rst]: It should be our company details.

File: report_hotel_reservation/README.rst [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Serpent Consulting                Services Pvt. Ltd.
```
Comment (Trivedi-Vacha-SerpentCS) [inline report_hotel_reservation/README.rst]: Remove in-between space

**Review by Trivedi-Vacha-SerpentCS — CHANGES_REQUESTED** [review]

**Comment by dhruvitaSerpentCS** [comment]
@Trivedi-Vacha-SerpentCS Done all suggested changes. Can you please review it?
Thanks.

File: report_hotel_reservation/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    "website": "http://www.serpentcs.com",
+    "depends": ["hotel_reservation"],
+    "license": "AGPL-3",
+    "category": "Generic Modules/Hotel Reservation",
+    "data": [
+        "security/ir.model.access.csv",
+        "views/report_hotel_reservation_view.xml",
+    ],
+    "installable": True,
+    "auto_install": False,
```
Comment (vimalpatelserpentcs) [inline report_hotel_reservation/__manifest__.py]: No need to define as false, by default it False

File: report_hotel_reservation/models/report_hotel_reservation.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+
+class ReportHotelReservationStatus(models.Model):
+
+    _name = "report.hotel.reservation.status"
+    _description = "Reservation By State"
+    _auto = False
+
+    reservation_no = fields.Char("Reservation No", readonly=True)
+    nbr = fields.Integer("Reservation", readonly=True)
```
Comment (vimalpatelserpentcs) [inline report_hotel_reservation/models/report_hotel_reservation.py]: Please change fields name `nbr -> no_of_reservation`

**Review by vimalpatelserpentcs — CHANGES_REQUESTED** [review]

**Review by vimalpatelserpentcs — APPROVED** [review]

**Review by Trivedi-Vacha-SerpentCS — APPROVED** [review]

**Pull request merged**